packages/auto-label/src/auto-label.ts

```diff
@@ -191,14 +199,18 @@ handler.addLabeltoRepoAndIssue = async function addLabeltoRepoAndIssue(
   return wasNotAdded;
 };
 
+function isFile(file: File | any): file is File {
```

Comment:
Please treat any gts warning as an error.  Since you're doing a cast below to `File`, and you're using the type guards, you can use `unknown` instead of `any` here without consequence :).

Reply:
Oh great, I was wondering how to address that. Thanks!